code/modules/world_api/commands/server_query.dm

```diff
@@ -21,12 +21,11 @@
 
 	s["players"] = clients.len
 	s["admins"] = 0
+	s["staff"] = staff.len
 
-	for(var/client/C in clients)
-		if(C.holder)
-			if(C.holder.fakekey)
-				continue
-
+	for(var/S in staff)
+		var/client/C = S
+		if(C.holder.rights & (R_MOD|R_ADMIN))
```

Comment:
I am pretty sure that this binary expression is wrong as that would only count people who have R_MOD and R_ADMIN

(R_MOD|R_ADMIN) results in a binary value containing both R_MOD and R_ADMIN

It should be sufficient to check for R_MOD as admins also have R_MOD

Reply:
Well me and Skull were not sure if admins have R_MOD. Plus I took it as all other code uses R_MOD|R_ADMIN

Reply:
Alberyk does not want retired admins to be counted, and they get both R_MOD and R_ADMIN, but not R_BAN which both admins and mods have.